tests/test_isrTaskLSST.py

```diff
@@ -477,6 +513,61 @@ def test_isrFlat(self):
         ratio = result2.exposure.image.array / result.exposure.image.array
         self.assertFloatsAlmostEqual(ratio, flat_nodefects.image.array, atol=1e-4)
 
+    def test_isrNoise(self):
+        """Test the recorded noise and gain in the metadata."""
+        mock_config = self.get_mock_config_no_signal()
+        # Remove the overscan scale so that the only variation
+        # in the overscan is from the read noise.
+        mock_config.overscanScale = 0.0
+
+        mock = isrMockLSST.IsrMockLSST(config=mock_config)
+        input_exp = mock.run()
+
+        isr_config = self.get_isr_config_electronic_corrections()
+        isr_config.doBias = True
+        # We do not do defect correction when processing biases.
+        isr_config.doDefect = False
+
+        # Need to make sure we are not masking the negative variance
+        # pixels when directly comparing calibration images and
+        # calibration-corrected calibrations.
+        isr_config.maskNegativeVariance = False
+
+        isr_task = IsrTaskLSST(config=isr_config)
+        with self.assertNoLogs(level=logging.WARNING):
+            result = isr_task.run(
+                input_exp.clone(),
+                bias=self.bias,
+                crosstalk=self.crosstalk,
+                ptc=self.ptc,
+                linearizer=self.linearizer,
+            )
+
+        metadata = result.exposure.metadata
+
+        for amp in self.detector:
+            # The overscan noise is always in adu and the readnoise is always
+            # in electron
+            gain = result.exposure.metadata[f"LSST ISR GAIN {amp.getName()}"]
+            read_noise = result.exposure.metadata[f"LSST ISR READNOISE {amp.getName()}"]
+
+            # Chcek that the gain and read noise are consistent with the
+            # values stored in the PTC
+            self.assertEqual(gain, self.ptc.gain[amp.getName()])
+            self.assertEqual(read_noise, self.ptc.noise[amp.getName()])
+
+            key = f"LSST ISR OVERSCAN RESIDUAL SERIAL STDEV {amp.getName()}"
+            self.assertIn(key, metadata)
+
+            # Determine if the residual serial overscan stddev is consistent
+            # with the PTC readnoise within 3xstandard error
+            serial_overscan_area = amp.getRawHorizontalOverscanBBox().area
```

Comment:
If we're doing MEDIAN_BY_ROW it'll be noisier than this, but this is a reasonable scaling.